apis/v1alpha1/gateway_types.go

```diff
@@ -234,6 +234,39 @@ type Listener struct {
 	//
 	// Support: Core
 	Routes RouteBindingSelector `json:"routes"`
+
+	// RouteTo defines how to route requests directly to a backend without
+	// creating an xRoute resource.
+	//
+	// If the referent cannot be found, the controller should raise
+	// the "ResolvedRefs" condition on the Gateway with the "DegradedListeners"
+	// reason. The gateway status should be updated with a condition
+	// that describes the error more specifically.
+	//
+	// Support: Extended
+	//
+	// +optional
+	RouteTo *ListenerRouteTo `json:"routeTo,omitempty"`
+}
+
+// ListenerRouteTo defines how a Listener can forward traffic to
+// a service name & port directly.
+type ListenerRouteTo struct {
```

Comment:
I think it would also be good to include protocol here as a required field, and then conformance/support of this feature could be based on the protocol.